addons/web_editor/static/src/less/web_editor.ui.components.less

```diff
@@ -299,6 +316,20 @@ body .modal {
                     }
                 }
             }
+            // VIDEO TAB
+            #editor-media-video {
+                #video-preview {
+                    width: 100%;
+                    max-width: 450px;
+                    height: auto;
+                    margin: 0 auto;
+
+                    iframe {
+                        max-width: 100%;
+                        max-height: 100%;
+                    }
+                }
+            }
```

Comment:
You can remove this code in this branch then so that there will be no (less) conflicts ;)